packages/terra-docs/src/Notice.module.scss

```diff
@@ -78,106 +300,176 @@
+      }
+
+      table {
+        tr:nth-child(2n) {
+          background-color: var(--terra-docs-notice-maintenance-table-zebra-background-color, #f6f8fa);
+        }
+
+        th,
+        td {
+          border-color: var(--terra-docs-notice-maintenance-table-cell-border-color, #dfe2e5);
+        }
+      }
+      /* stylelint-enable selector-max-compound-selectors, max-nesting-depth */
+    }
+  }
+  
   .important {
     background-color: var(--terra-docs-notice-important-background-color, #e9f5e0);
     border-color: var(--terra-docs-notice-important-border-color, #d7edc8);
 
     .title {
       color: var(--terra-docs-notice-important-title-color, #2a4a04);
 
-      /* stylelint-disable-next-line selector-max-compound-selectors */
+      /* stylelint-disable-next-line selector-max-compound-selectors, max-nesting-depth */
       > span {
         border-color: var(--terra-docs-notice-important-title-border-color, #81dd44);
       }
     }
 
-    pre {
-      border-color: var(--terra-docs-notice-important-pre-border-color, #81dd44);
+    blockquote {
+      background-color: var(--terra-docs-notice-important-blockquote-background-color, rgba(215, 237, 200, 0.6));
+      border-color: var(--terra-docs-notice-important-blockquote-border-color, #81dd44);
     }
-  }
 
-  .accessory {
-    background-position: 0 0;
-    background-repeat: no-repeat;
-    background-size: auto;
-    float: left;
-    height: 0;
-    margin: 0;
-    vertical-align: var(--terra-docs-notice-accessory-vertical-align, -0.15rem);
-    width: 0;
-    @media screen and (-ms-high-contrast: active), (-ms-high-contrast: none) {
-      vertical-align: var(--terra-docs-notice-accessory-ms-vertical-align, -0.15rem);
+    code {
+      background-color: var(--terra-docs-notice-important-code-background-color, #cfebbc);
+    }
+
+    hr {
+      background-color: var(--terra-docs-notice-important-hr-background-color, #81dd44);
     }
+
+    .children {
+      /* stylelint-disable selector-max-compound-selectors, max-nesting-depth */
+      pre {
+        border-color: var(--terra-docs-notice-important-pre-border-color, #81dd44);
+        box-shadow: var(--terra-docs-notice-important-pre-box-shadow, 0 5px 2px -3px rgba(0, 110, 13, 0.16), 2px 4px 5px 2px rgba(0, 110, 13, 0.08));
+      }
+
+      table {
+        tr:nth-child(2n) {
+          background-color: var(--terra-docs-notice-important-table-zebra-background-color, rgba(215, 237, 200, 0.15));
+        }
+
+        th,
+        td {
+          border-color: var(--terra-docs-notice-important-table-cell-border-color, #bce79f);
+        }
+      }
+      /* stylelint-enable selector-max-compound-selectors, max-nesting-depth */
+    }    
   }
 
-  .title {
-    font-size: 0.92857rem;
-    font-weight: 600;
-    line-height: 1.38461;
-    margin: 0;
-    padding: 0;
-    text-transform: uppercase;
+  .not-supported {
+    background-color: var(--terra-docs-notice-not-supported-background-color, #fff);
```

Comment:
Other notices use a non-white background color. Is the use of a white background-color intentional for the not-supported notice?

Reply:
It was intentional to not have it have a background color so it would look "quieter", only using the icon and the left bar color to direct attention, but not too much attention. 

Visually it lets the others stand out more (caution, important, etc.), and this warning can be present along with those without being a red flashing siren each time we use it.

![Jerry Seinfeld rubbing eyes from bright red light](https://c.tenor.com/BPG_ZX7m8iUAAAAM/burning-brain-seinfeld.gif)

Reply:
While this PR is still open, you can see how it works with the other notices types on the same page for Pills:
- [PR#1521.../cerner-terra-framework-docs/pills/filter-pills#incorrect-use](https://terra-framew-terra-filt-zvasyv.herokuapp.com/components/cerner-terra-framework-docs/pills/filter-pills#incorrect-use)
- [PR#1521.../cerner-terra-framework-docs/pills/accessibility-guide](https://terra-framew-terra-filt-zvasyv.herokuapp.com/components/cerner-terra-framework-docs/pills/accessibility-guide)